### PR TITLE
Feature/COM-3264 Integrated Shipping Method API's

### PR DESCRIPTION
**Pull request by hrishid**

### CODE REVIEW

File: scripts/modules/b2b-account/edit-quote.js [hunk 1]
```diff
@@ -873,28 +878,92 @@ define([
                 }
                 this.model.set('auditHistory', auditHistory);
             }
-        },
+        },    
+   
+        getAvailableShippingMethods: function()
+        { var self = this;
```
Comment (rajeshdatir) [inline scripts/modules/b2b-account/edit-quote.js:884]: add a variable on new line

File: scripts/modules/b2b-account/edit-quote.js [hunk 2]
```diff
@@ -873,28 +878,92 @@ define([
...
+            var json = JSON.parse(JSON.stringify(
+                {
+                       "quoteId": self.model.get('id'),
+                       "draft": true 
+
```
Comment (ghost) [inline scripts/modules/b2b-account/edit-quote.js]: Remove blank lines

File: scripts/modules/b2b-account/edit-quote.js [hunk 3]
```diff
@@ -873,28 +878,92 @@ define([
...
+                }));                              
+          return self.model.apiModel.getAvailableShippingMethods(json).then(function (response) {
+            self.model.set('shippingMethods', response);
+            self.render();            
+            }, function (error) {
+            self.showMessageBar(error);
+        });         
+       },
+       shippingMethodSelectionChanged: function()
+       {    var self = this;
```
Comment (ghost) [inline scripts/modules/b2b-account/edit-quote.js]: add curly braces after _function() {_. make it consistent with other functions

**Review by ghost — CHANGES_REQUESTED** [review]

File: scripts/modules/b2b-account/edit-quote.js [hunk 4]
```diff
@@ -873,28 +878,93 @@ define([
...
+        updateFulfillmentInfo: function (updateMode) {
+            var self = this;
+            updateMode = updateMode || applyToDraft;     
+            self.getAvailableShippingMethods();     
+            self.model.set('updatemode', updateMode);
+            self.model.isLoading(true);
+            var fulfillmentInfo = self.model.get("fulfillmentInfo");
+            if(fulfillmentInfo)
+            {
+            var json = JSON.parse(JSON.stringify(
```
Comment (rakesh-dontula) [inline scripts/modules/b2b-account/edit-quote.js]: indent properly. Use some formatter to format the code properly.

**Review by rakesh-dontula — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
As part of my rebase I reverted a no-op change to `templates/modules/multi-ship-checkout/shipping-method-item.hypr.live` as it wasn't needed. Cleaned up some whitespace/formatting issues as well. No functional changes.

**Pull request merged**